### PR TITLE
updates cross-env to latest (from prior: August 2016 version)

### PR DESCRIPTION
**Pull request by dimitropoulos**
This could use an update.  From looking at the release notes there are no changes that affect our use case.  It turns out this library has been officially abandoned: https://github.com/kentcdodds/cross-env/issues/257, but it can't hurt to be on a semi-recent version.

It's hard to reverse the clock now, but I'm nearly sure I hit a problem with cross-env once (in insomnia-app) and I wonder, now having learned how out of date it is, if this was the reason.

### CODE REVIEW

**Comment by jackkav** [comment]
I'm curious what the tradeoffs would be if we remove TS_NODE_PROJECT and make the default base ES5/ESNext and module commonjs.
https://webpack.js.org/configuration/configuration-languages/#typescript

**Comment by dimitropoulos** [comment]
@jackkav this PR is just a simple (nearly) noop dependency update.  would you mind if we took the conversation about our _usage_ of cross-env to a separate PR or discussion somewhere?

File: packages/insomnia-app/package.json [hunk 1]
```diff
@@ -228,7 +228,7 @@
     "@types/webpack-env": "^1.16.0",
     "@types/yaml": "^1.9.7",
     "concurrently": "^7.0.0",
-    "cross-env": "^2.0.0",
+    "cross-env": "^7.0.3",
```
Comment (DMarby) [inline packages/insomnia-app/package.json:232]: We don't seem to be actually using `cross-env` in this package, so probably makes more sense to get rid of it entirely here rather then updating it?

Comment (dimitropoulos) [reply packages/insomnia-app/package.json:232]: oh, sweet.  I didn't realize.  I think you meant to put this comment on the root package.json (insomnia-app is using it quite a bit), but if I misunderstood something, please unresolve.

updated in 6cb543b4692be9e67855efb5e9f948bb56c27fb9

Comment (DMarby) [reply packages/insomnia-app/package.json:232]: My bad, did indeed mean to put it in the root package.json, yep!

**Review by DMarby — APPROVED** [review]
:shipit:

**Pull request merged**